src/modals/confirmationModal/container.jsx

```diff
@@ -10,15 +10,15 @@ import checkFileErrors from '../../utils/checkFileErrors.js'
 import ConfirmationModal from './index.jsx'
 
 export function mapStateToProps(state) {
-  const id = state.app.institutionId
+  const lei = state.app.lei
   const { showing } = state.app.confirmation
   const { filingPeriod } = state.app
   const { code } = state.app.submission.status
   const { newFile } =
-    state.app.upload[id] || state.app.upload['__DEFAULT_UPLOAD__']
+    state.app.upload[lei] || state.app.upload['__DEFAULT_UPLOAD__']
 
   return {
-    id,
+    lei,
```

Comment:
This `lei` update requires changes to [`confirmationModal/index.jsx`](https://github.com/cfpb/hmda-platform-ui/blob/393ab2cc9497be3e5345f328eb99f9f74ed61f7b/src/modals/confirmationModal/index.jsx#L49) too.

Reply:
Its causing

```
TypeError: Cannot read property 'focus' of undefined
ModalConfirm._focusButton
src/modals/confirmationModal/index.jsx:8
   5 | import '../Modal.css'
   6 | 
   7 | let _focusButton = function() {
>  8 |   return this.confirmButton.focus()
   9 | }
  10 | 
  11 | let _focusLink = function() {
```